README.md

```diff
@@ -131,22 +131,84 @@ Your Public and Signing Keys are saved in:
 * [Go v1.20+](https://go.dev/dl/)
 
 
-Clone source code
+Clone source code into desired folder. For this, you have to access the desired folder and run the following command:
 
-    mkdir ~/go/src/github.com/Coiin-Blockchain
-    cd ~/go/src/github.com/Coiin-Blockchain
     git clone https://github.com/Coiin-Blockchain/nvl-independent-signer.git
 
 Build
-    cd ~/go/src/github.com/Coiin-Blockchain/nvl-independent-signer
-    go mod tidy
-    go build
+    Access the folder containing the main.go file
+    execute the command 'go mod tidy'
+    execute the command 'go build'
 
 Run
    go run main.go
```

Comment:
We need to change this lines as well. The procedure to run this script requires to run `make all` to create the scripts required to every platform or `make ${platform}`, where `${platform}` can be anything in the list [windows, linux, darwin, raspberry]